### PR TITLE
Allow popup height to be changed with a setter

### PR DESCRIPTION
**Pull request by VitalyKuznetsov**
Adding functionality to EmojiPopup to be opened in a different mode:
Keyboard height is not dynamically calculated, but set manually instead.

### CODE REVIEW

**Comment by mario** [comment]
I'd prefer the approach that was mentioned earlier - allow setting height in the builder and if it's 0, calculate the height - otherwise use set height.

**Comment by VitalyKuznetsov** [comment]
Consider the following case:

1. User opens app and uses emojiPopup -> height is set in builder.
2. User changes keyboard height in settings.
3. Goes back to the app, emojiPopup has not been destroyed yet.
4. Upon keyboard opening we save its height, however emojiPopup height cannot be changed, until a new instance is created.

For such scenario a simple setter would be quite useful.

**Comment by VitalyKuznetsov** [comment]
So what do you think?

**Comment by VitalyKuznetsov** [comment]
Hey, @mario , take a look!
Maybe it is more like what you are looking for?

File: emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java [hunk 1]
```diff
@@ -398,6 +432,9 @@ private Builder(final View rootView) {
       emojiPopup.onEmojiPopupShownListener = onEmojiPopupShownListener;
       emojiPopup.onEmojiPopupDismissListener = onEmojiPopupDismissListener;
       emojiPopup.onEmojiBackspaceClickListener = onEmojiBackspaceClickListener;
+      if (this.popupWindowHeight > 0) {
```
Comment (vanniktech) [inline emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java]:
```suggestion
      if (popupWindowHeight > 0) {
```

**Comment by VitalyKuznetsov** [comment]
Yeah sure, but the gboard has an option to change its height from settings, which may occure when emojipopup is already created.
In this case I'd like to change its height without creating a new instance.

**Comment by mario** [comment]
I'll test this in the coming days.

**Comment by VitalyKuznetsov** [comment]
Any updates? @mario

**Comment by mario** [comment]
Looks ok.

**Comment by vanniktech** [comment]
@VitalyKuznetsov can you fix the CI?

**Comment by mario** [comment]
Can be merged after rebase.

**Comment by vanniktech** [comment]
@VitalyKuznetsov are you still using this method and if so; can you show me your code which detects that the keyboard height has been changed and hence you are manually calling this method?

**Pull request merged**